docs/user-guides/ca-based-issuer.md

```diff
@@ -0,0 +1,57 @@
+# Creating a simple CA based issuer
+
+`cert-manager` can be used to obtain certificates using a signing key pair. There are various tools you can use to generate your own key pair including [openssl][1] and [cfssl][2]. To generate a signing key pair using openssl, run the following commands:
+
+* `openssl genrsa -out ca.key 2048`
+* `openssl req -x509 -new -nodes -key ca.key -subj "/CN=${COMMON_NAME}" -days 10000 -out ca.crt`
```

Comment:
Can we set `days` to `3650` (10yrs) and make a note that this is for a quick and basic CA.